sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -253,6 +253,7 @@ final class DataFrameWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * (scala-specific)
```

Comment:
Nit: I don't think this is necessary. It's in the Scala API and technically callable from Java. Call the other one Java-friendly.